client/src/components/shared/Layout/LayoutTopBar/LayoutTopBar.tsx

```diff
@@ -239,14 +244,14 @@ const LayoutTopBar: FC<LayoutTopBarProps> = ({ className }) => {
         <>
           <Drawer
             dataTest="SettingsDrawer"
-            isOpen={isSettingsDrawerOpen}
+            isOpen={isSettingsDrawerOpen && !isTimeoutListPresenceModalOpen?.value}
```

Comment:
Why? Drawer can't be open below the modal?

Reply:
It can be, but in my opinion it's a better solution (drawer will show when Sybil modal is closed).